vmaas_sync/pkg_sync.go

```diff
@@ -45,35 +45,27 @@ func storePackageNames(tx *gorm.DB, pkgs map[string]vmaas.PackagesResponsePackag
 		nameMap[nevra.Name] = true
 		byNevra[*nevra] = detail
 	}
-	nameArr := make([]string, 0, len(nameMap))
+	var nameArr []string
+	var pkgNames []models.PackageName
 	for n := range nameMap {
 		nameArr = append(nameArr, n)
+		pkgNames = append(pkgNames, models.PackageName{Name: n})
 	}
-
-	var pkgNames []models.PackageName
-	err := tx.Set("gorm:query_option", "FOR UPDATE OF package_name").
-		Where("name in (?)", nameArr).Find(&pkgNames).Error
+	// Insert missing
+	tx = tx.Set("gorm:insert_option", "ON CONFLICT DO NOTHING")
+	errs := database.BulkInsertChunk(tx, pkgNames, 2048)
```

Comment:
What is 2048 constant? It should be declared as a constant and then used via constant name.